code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -652,21 +771,88 @@ Congratulations! You are now trained for invasive xenobiology research!"}
 	origin_tech = "materials=2;biotech=2;abductor=2"
 	toolspeed = 0.25
 
-/obj/item/clothing/head/helmet/abductor
-	name = "agent headgear"
-	desc = "Abduct with style - spiky style. Prevents digital tracking."
-	icon_state = "alienhelmet"
-	item_state = "alienhelmet"
-	blockTracking = 1
-	origin_tech = "materials=7;magnets=4;abductor=3"
-	flags = BLOCKHAIR
-	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE
+/////////////////////////////////////////
+//////////// JANITORIAL TOOLS ///////////
+/////////////////////////////////////////
+/obj/item/mop/advanced/abductor
+	name = "alien mop"
+	desc = "A collapsable mop clearly used by aliens to clean up any evidence of a close encounter. The head produces a constant supply of water when run over a surface, seemingly out of nowhere."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "mop_abductor"
+	mopcap = 100
+	origin_tech = "materials=3;engineering=3;abductor=3"
+	refill_rate = 50
+	refill_reagent = "water"
+	mopspeed = 10
+
+/obj/item/soap/syndie/abductor
+	name = "alien soap"
+	desc = "Even bloodless aliens need to wash the grime off. Smells like gunpowder."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "soap_abductor"
 
-	sprite_sheets = list(
-		"Vox" = 'icons/mob/clothing/species/vox/head.dmi'
+/obj/item/lightreplacer/bluespace/abductor
+	name = "alien light replacer"
+	desc = "It's important to keep all the mysterious lights on a UFO functional when flying over backwater country."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "lightreplacer_abductor"
+	origin_tech = "magnets=3;engineering=4;abductor=3"
+	max_uses = 40
+	uses = 20
+
+/obj/item/melee/flyswatter/abductor
+	name = "alien flyswatter"
+	desc = "For killing alien insects, obviously."
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "flyswatter_abductor"
+	item_state = "flyswatter_abductor"
+	origin_tech = "abductor=1"
+	force = 2 // Twice as powerful thanks to alien technology!
+	throwforce = 2
+
+/obj/item/reagent_containers/spray/cleaner/safety/abductor	// Essentially an Advanced Space Cleaner, but abductor-themed. For the implant.
```

Comment:
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/5d95db88-b85f-4b4b-888d-f6c7242dfee3)

For some reason its not using the correct mob sprite.

Reply:
Fixed it.